### ISSUE TITLE
:bomb: `Bug`: dtm init failed in version 0.6.1

### ISSUE THREAD

**Issue by zhujinhe** [devstream-io/devstream#695]
### What Happened?

I'm following the tutorial `https://docs.devstream.io/en/latest/quickstart/` step by step.
and failed in step 3: dtm init -f quickstart.yaml
here is the error message:

```
# dtm init -f quickstart.yaml
2022-06-20 12:29:21 ✖ [ERROR]  Config file validation failed: tool file is empty.
2022-06-20 12:29:21 ℹ [INFO]  Using dir <.devstream> to store plugins.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11e8537]

goroutine 1 [running]:
github.com/devstream-io/devstream/internal/pkg/pluginmanager.DownloadPlugins(0x0)
	github.com/devstream-io/devstream/internal/pkg/pluginmanager/manager.go:30 +0x177
main.initCMDFunc(0x18a8a80, {0xc000045cc0, 0x0, 0x2})
	github.com/devstream-io/devstream/cmd/devstream/init.go:26 +0x185
github.com/spf13/cobra.(*Command).execute(0x18a8a80, {0xc000045ca0, 0x2, 0x2})
	github.com/spf13/cobra@v1.4.0/command.go:860 +0xb67
github.com/spf13/cobra.(*Command).ExecuteC(0x18a9200)
	github.com/spf13/cobra@v1.4.0/command.go:974 +0x8b3
github.com/spf13/cobra.(*Command).Execute(0x18a9200)
	github.com/spf13/cobra@v1.4.0/command.go:902 +0x2f
main.main()
	github.com/devstream-io/devstream/cmd/devstream/main.go:98 +0x32
```

It seems that setting toolFile to "" was not allowed in version 0.6.1.
https://github.com/devstream-io/devstream/blob/adf3dffa19d2450a367f998717a1a11c5e842cc8/internal/pkg/configloader/validation.go#L85-L87

### Anything else

dtm version
```0.6.1```

Nothing changed in  quickstart.yaml except the YOUR_GITHUB_USERNAME_CASE_SENSITIVE and YOUR_DOCKER_USERNAME by the tutorial:
```
---
# core config
varFile: "" # If not empty, use the specified external variables config file
toolFile: "" # If not empty, use the specified external tools config file
state: # state config, backend can be local or s3
  backend: local
  options:
    stateFile: devstream.state

---
# plugins config
tools:
  - name: github-repo-scaffolding-golang
    instanceID: default
    options:
      owner: zhujinhe
      repo: go-webapp-devstream-demo
      branch: main
      image_repo: zhujinhe/go-webapp-devstream-demo
  - name: githubactions-golang
    instanceID: default
    dependsOn: ["github-repo-scaffolding-golang.default"]
    options:
      owner: zhujinhe
      repo: go-webapp-devstream-demo
      language:
        name: go
        version: "1.17"
      branch: main
      build:
        enable: True
      test:
        enable: True
        coverage:
          enable: True
      docker:
        enable: False
```

### DevStream Version

latest

**Comment by imxw** [comment]
The problem has been resolved by #655. However, version 0.6.1 does not yet include this change. Please refer to https://docs.devstream.io/en/v0.6.1/quickstart_en/ for now. @zhujinhe

**Comment by daniel-hutao** [comment]
![image](https://user-images.githubusercontent.com/18692628/174609560-85e697e2-bf7d-4176-b9f9-9b1d9c03227f.png)

By default, the documentation version is displayed as latest, so there are some inconsistencies with `dtm` version `0.6.1`. Please switch to version 0.6.1 of the documentation and try again. @zhujinhe 

cc @aFlyBird0 @IronCore864

**Comment by hxcGit** [comment]
@xiaozhi747 Did you download the correct version of dtm? For mac-m1-arm64, please use https://devstream.gateway.scarf.sh/releases/v0.7.0/dtm-darwin-arm64. I've tested locally and everything works fine.

**Comment by IronCore864** [comment]
@zhujinhe we did some changes regarding documentation and releases. To sum up:

- We follow [this doc](https://docs.devstream.io/en/latest/development/branch-and-release/) for releases and branches. TL;DR: we will try our best to support the latest 3 versions. Given our current shortage of manpower, this can't be guaranteed at the moment, but we will strive for it.
- Documentation versions: now that the v0.7 is released, we made the v0.7 version the default version when you open docs.devstream.io. The "latest" version won't be shown as default anymore.

If you are interested, you can give the quickstart a try according to the v0.7 documentation. Let us know if there are any more issues.

Thanks very much for your contribution to this matter. Closing this issue for now, but do re-open or create a new one if you find more issues according to the v0.7 doc.
